### DISCUSSION TITLE
Dynamic thresholding

### DISCUSSION THREAD

**Discussion by gxmikvid** [WASasquatch/was-node-suite-comfyui#62] [Ideas]
I made some custom nodes before but the feature described below is above my level. So here I am.

An option in the sampler nodes, or even a separate node to enable us to use high cfg without the burn-in effect.

Along the lines of:
https://github.com/mcmonkeyprojects/sd-dynamic-thresholding

Great work btw. Made my life easier.

**Comment by WASasquatch** [comment]
I think Comfyanonymous is working on something for this. I couldn't find the topic where it was mentioned but they specifically mentioned working on something to allow higher CFG.

**Reply by gxmikvid** [reply]
I'll look into it. Thank you.
In the mean time, for anyone that might stumble upon this, I found that unipc with 8 cfg works as a substitute.

**Comment by bobpuffer1** [comment]
Comfyanonymous did finish that custom node still experimental but I have it working and it's working well
